### PR TITLE
fix: [Sign & submit] Content updates and bug fixes

### LINKED ISSUE #722: [Sign & Submit] Design and content updates
- [ ] Submit: Success alert content
- [ ] Submit: Spacing above `Update your parent entity`

### PR DESCRIPTION
**Pull request by meissadia**
Close #722 

## Changes
- Task: Update top warning alert content
- Task: Update bottom success alert content
- Fix: Source for Warning/Error counts

## How to review
Compare against the design file

## Top warning alert
|Time|Screenshot|
|---|---|
|Before|Screenshot 2024-06-18 at 9 56 10 AM|
|After|Screenshot 2024-06-18 at 9 56 38 AM|

## Bottom success alert
Screenshot 2024-06-17 at 2 01 33 PM

### CODE REVIEW

**Comment by natalia-fitzgerald** [comment]
@meissadia - Looks good to me!

**Comment by meissadia** [comment]
Added fix for source of warnings/error counts.

File: src/types/filingTypes.ts [hunk 1]
```diff
@@ -143,8 +143,11 @@ export interface ValidationResults {
 }
 
 export interface ValidationErrorWarning {
-  count: number;
   details: Detail[];
+  multi_field_count: number;
```
Comment (billhimmelsbach) [inline src/types/filingTypes.ts:147]: I think some of these changes are taken care of by @shindigira's PR already [over here](https://github.com/cfpb/sbl-frontend/pull/699/files#diff-580e104f64a806e9ed0e66e14eec71aa0d3f851a2ca9d7650ea3a14e4f63d28aR146-R149). I don't this his PR fixes the count on the submission page though. He didn't get a chance to merge in his PR before he left on vacation, so I'm going to go ahead and merge it now and you can update this branch and see what changes still need to be made @meissadia?

**Review by billhimmelsbach — DISMISSED** [review]
Ok, I merged in [Sigmund's PR](https://github.com/cfpb/sbl-frontend/pull/699/files#diff-580e104f64a806e9ed0e66e14eec71aa0d3f851a2ca9d7650ea3a14e4f63d28aR146-R149), so now just resolve the conflict in `filingTypes.ts` (I think just removing your changes) and you're good to go.

**Review by billhimmelsbach — APPROVED** [review]
Sweet!

**Pull request merged**